modules/claims_api/app/models/claims_api/veteran.rb

```diff
@@ -28,5 +29,31 @@ def va_profile=(new_va_profile)
       raise Common::Exceptions::ParameterMissing 'X-VA-Birth-Date' unless matches
       super(new_va_profile)
     end
+
+    def self.from_headers(headers, with_gender = false)
```

Comment:
Making `with_gender` a keyword arg would eliminate the ambiguous boolean issue where it is called. E.g.

```
Veteran.from_headers(headers, true)
```

versus

```
Veteran.from_headers(headers, with_gender: true)
```

Reply:
I dig it, will update.

Reply:
resolved in fd76477